pkg/universe.dagger.io/docker/test/image.cue

```diff
@@ -65,29 +65,21 @@ dagger.#Plan & {
 				input: myimage
 				command: name: "ls"
 				export: files: {
-					"/dagger.txt": _ & {
-						contents: "not hello from dagger"
-					}
-					"/bin/dagger": _ & {
-						contents: """
-							#!/bin/sh
-							echo -n "hello from $HELLO_FROM" > /dagger.txt
+					"/dagger.txt": string & "not hello from dagger"
+					"/bin/dagger": string & """
+						#!/bin/sh
+						echo -n "hello from $HELLO_FROM" > /dagger.txt
 
-							"""
-					}
+						"""
 				}
 			}
 			verify_cmd_is_run: docker.#Run & {
 				input: myimage
-				export: files: "/dagger.txt": _ & {
-					contents: "hello from dagger"
-				}
+				export: files: "/dagger.txt": string & "hello from dagger"
```

Comment:
Doesn't just `export: files: "/dagger.txt": "hello from dagger"` work?

Reply:
Yes it does! I'll fix it right now

Reply:
Fixed in https://github.com/dagger/dagger/pull/2524/commits/1f224c980f828a58967f484cfc376f751722fe98